pages/common/ffplay.md

```diff
@@ -7,6 +7,14 @@
 
 `ffplay {{path/to/file}}`
 
+- Play audio from a media file without a GUI:
+
+`ffplay -nodisp {{path/to/file}}`
+
+- Play media passed by `ffmpeg` through standard stream:
+
+`ffmpeg -i {{path/to/file}} -c {{copy}} -f {{mediaformat}} - | ffplay -`
```

Comment:
```suggestion
`ffmpeg -i {{path/to/file}} -c {{copy}} -f {{media_format}} - | ffplay -`
```
I would suggest using the snake case here as mentioned in [style guide](https://github.com/tldr-pages/tldr/blob/main/contributing-guides/style-guide.md#naming)